server/server.js

```diff
@@ -12,6 +12,7 @@ const artistHandler = require('./controllers/artist');
 const imageHandler = require('./controllers/imageUpload');
 const notificationHandler = require('./controllers/notification');
 const app = express();
+const worker = require('./worker.js');
```

Comment:
I don't know if we need to require this. I think that the worker can run in it's own node process as long as it can connect to the database.